extensions/mssql/src/objectManagement/ui/loginDialog.ts

```diff
@@ -95,31 +95,27 @@ export class LoginDialog extends ObjectManagementDialogBase<ObjectManagement.Log
 	}
 
 	protected async initializeUI(): Promise<void> {
-		this.dialogObject.registerContent(async view => {
-			const sections: azdata.Component[] = [];
-			this.initializeGeneralSection(view);
-			sections.push(this.generalSection);
-
-			if (this.isNewObject || this.objectInfo.authenticationType === 'Sql') {
-				this.initializeSqlAuthSection(view);
-				sections.push(this.sqlAuthSection);
-			}
+		const sections: azdata.Component[] = [];
+		this.initializeGeneralSection();
+		sections.push(this.generalSection);
 
-			this.initializeServerRolesSection(view);
-			sections.push(this.serverRoleSection);
+		if (this.isNewObject || this.objectInfo.authenticationType === 'Sql') {
```

Comment:
We have https://github.com/Microsoft/azuredatastudio/blob/b7e0bbb64a565fd771439403fde9812cc7817fe3/src/sql/azdata.proposed.d.ts#L523 which we should be generally using - but why is the auth type "sql" here and not "SqlLogin"?

Reply:
@caohai could you please update STS side to use the strings consistent with the enums defined azdata typings file?

Reply:
@alanrenmsft Do you mean make the following string changes?
Sql -> SqlLogin
Windows -> Integrated
AAD -> AzureMFA/AzureMFAAndUser

The first two make sense but for AAD, we don't need to distinguish between MFA/MFAAndUser...

Reply:
I guess you can leave AAD as is.

Reply:
Sounds like you'd probably want to make a separate enum then if it's not a direct 1-1 mapping.